specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ManagedEnvironmentsDaprResiliencyRetryPolicies.json

```diff
@@ -0,0 +1,282 @@
+            "required": true,
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprResiliencyRetryPolicy"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DaprResiliencyRetryPolicy"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or update Dapr Resiliency Retry Policy": {
+            "$ref": "./examples/DaprResiliencyRetryPolicies_CreateOrUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "DaprResiliencyRetryPolicies"
+        ],
+        "summary": "Delete a Dapr Resiliency Retry Policy.",
+        "description": "Delete a Dapr Resiliency Retry Policy from a Managed Environment.",
+        "operationId": "DaprResiliencyRetryPolicies_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Managed Environment.",
+            "required": true,
+            "pattern": "^.*",
+            "type": "string"
+          },
+          {
+            "name": "policyName",
+            "in": "path",
+            "description": "Name of the Dapr Resiliency Retry Policy.",
+            "required": true,
+            "pattern": "^.*",
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Delete operation completed"
+          },
+          "204": {
+            "description": "Environment does not exist"
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete Dapr Resiliency Retry Policy": {
+            "$ref": "./examples/DaprResiliencyRetryPolicies_Delete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DaprResiliencyRetryPoliciesCollection": {
```

Comment:
>DaprResiliencyRetryPoliciesCollection [](http://example.com/codeflow?start=5&length=37)

I think you have already defined this in your commondefinitions file

Reply:
I had to add the definitions here too otherwise the automation failed.

Reply:
Duplicate definitions need to be fixed. Either remove them from the CommonDefintions.json or refer to the object from CommonDefintions file instead of defining it again in each file.